src/index.js

```diff
@@ -1,277 +1,234 @@
+    message: 'Migrate Percy config file?',
+    default: true
+  }]);
 
-    let { doConfig } = await inquirer.prompt([{
-      type: 'confirm',
-      name: 'doConfig',
-      message: 'Migrate Percy config file?',
-      default: true
-    }]);
+  if (doConfig) {
+    let percybin = `${process.cwd()}/node_modules/@percy/cli/bin/run`;
 
-    if (doConfig) {
-      let percybin = `${process.cwd()}/node_modules/@percy/cli/bin/run`;
-
-      if (!existsSync(percybin)) {
-        this.log.warn('Could not run config migration, @percy/cli is not installed');
-      } else {
-        await run(percybin, ['config:migrate']);
-      }
+    if (!fs.existsSync(percybin)) {
+      log.warn('Could not run config migration, @percy/cli is not installed');
+    } else {
+      await run(percybin, ['config:migrate']);
     }
   }
+}
 
-  // Confirms if the first SDK in the list is the current SDK, otherwise will present a list of
-  // supported SDKs to choose from, erroring when the chosen SDK is not in the list
-  async confirmSDK({ installed, inspected }, name) {
-    let sdk;
-
-    let fromInstalled = SDK => {
-      let sdk = installed.find(sdk => sdk instanceof SDK) || new SDK();
-
-      if (!sdk.installed && inspected.includes(sdk.language)) {
-        this.log.warn('The specified SDK was not found in your dependencies');
-      }
+// Confirms if the first SDK in the list is the current SDK, otherwise will present a list of
+// supported SDKs to choose from, erroring when the chosen SDK is not in the list
+async function confirmSDK({ installed, inspected }, name, log) {
+  let sdk;
 
-      return sdk;
-    };
+  let fromInstalled = SDK => {
+    let sdk = installed.find(sdk => sdk instanceof SDK) || new SDK();
 
-    // don't guess when a name is provided
-    if (name) {
-      let SDK = migrations.find(SDK => SDK.matches(name));
-      if (!SDK) this.log.warn('The specified SDK is not supported');
-      else sdk = fromInstalled(SDK);
-    } else {
-      [sdk] = installed;
+    if (!sdk.installed && inspected.includes(sdk.language)) {
+      log.warn('The specified SDK was not found in your dependencies');
     }
 
-    // confirm the specified or guessed sdk
-    if (sdk) {
-      let { isSDK } = await inquirer.prompt([{
-        type: 'confirm',
-        name: 'isSDK',
-        message: `Are you currently using ${sdk.aliased}?`,
-        default: true
-      }]);
-
-      if (isSDK) {
-        return sdk;
-      }
-    }
+    return sdk;
+  };
 
-    // ask to choose from list of supported sdks
-    let { fromChoice } = await inquirer.prompt([{
-      type: 'list',
-      name: 'fromChoice',
-      message: 'Which SDK are you using?',
-      default: sdk ? migrations.indexOf(sdk.constructor) : 0,
-      choices: migrations.map(SDK => ({
-        name: SDK.aliased,
-        value: () => fromInstalled(SDK)
-      })).concat({
-        name: '...not listed?',
-        value: null
-      })
-    }]);
+  let migrations = await migration.load();
```

Comment:
Seems odd that `migration.load` (singular) returns an array of multiple migrations. Perhaps another name would fit better?

What about making the util `migrations` and renaming these variabes to `sdks`? This would also fit with the code immediately under this.

```javascript
let SDK = sdks.find(SDK => SDK.matches(name));
...
```